README.md

```diff
@@ -1,11 +1,11 @@
 # PR Metrics
 
-PR Metrics is a both a GitHub Action and [Azure DevOps/Pipelines][azuredevops]
+PR Metrics is a both a GitHub Action and [Azure DevOps Pipelines][azuredevops]
 task for adding size and test coverage indicators to the start of each Pull
 Request title.
 
-The Azure Pipelines task can be downloaded from the Visual Studio Marketplace at
-<https://aka.ms/PRMetrics>.
+The Azure Pipelines task can be downloaded from the
+[Visual Studio Marketplace][vsmarketplace].
```

Comment:
What does this mean when you say it can be downloaded at [Visual Studio Marketplace][vsmarketplace]?

Reply:
Is that a link?